app/assets/stylesheets/theme/_autocomplete.scss

```diff
@@ -0,0 +1,101 @@
+.easy-autocomplete {
+  position: relative;
+}
+.easy-autocomplete input {
+  border-color: #ccc;
```

Comment:
Good call @adarsh. Actually, taking it a step further, we should either use colors defined in the [`_variables.scss`](https://github.com/greencommons/commons/blob/master/app/assets/stylesheets/theme/_variables.scss), or add something to it. For this, maybe one of the `$gray-*` colors?